NTupleMaker/interface/functionsCP.h

```diff
@@ -795,60 +850,67 @@ if(Pi1[1]==Pi2[1]&&Pi1[1]!=0)cout<<"BIG  strange issue, we're using one vector f
   //  std::cout << "Ux(1) = " << ref1.Px() << "  Uy(1) = " << ref1.Py() << "  Uz(1) = " << ref1.Pz() << std::endl;
   //  std::cout << "Ux(2) = " << ref2.Px() << "  Uy(2) = " << ref2.Py() << "  Uz(2) = " << ref2.Pz() << std::endl;
   //  std::cout << std::endl;
+  
   // get 3-vectors
   TVector3 vecPi1 = Pi1.Vect();
   TVector3 vecPi2 = Pi2.Vect();
   TVector3 vecRef1 = ref1.Vect();
   TVector3 vecRef2 = ref2.Vect();
 
-  // normalize them 
+  // normalize them
+  
   vecPi1 *= 1/vecPi1.Mag();
   vecPi2 *= 1/vecPi2.Mag();
   vecRef1 *= 1/vecRef1.Mag();
   vecRef2 *= 1/vecRef2.Mag();
-
+    
   // transverse components  
   TVector3 vecRef1transv = vecRef1 - vecPi1*(vecPi1*vecRef1);
   TVector3 vecRef2transv = vecRef2 - vecPi2*(vecPi2*vecRef2);
-  
-  /*
-    cout<<"vecRef1transv.Mag() "<<vecRef1transv.Mag() <<endl;
-    cout<<"vecRef2transv.Mag() "<<vecRef2transv.Mag() <<endl;
-    cout<<"vecRef1transv[0] "<<vecRef1transv[0] <<endl;
-    cout<<"vecRef1transv[1] "<<vecRef1transv[1] <<endl;
-    cout<<"vecRef1transv[2] "<<vecRef1transv[2] <<endl;
-    cout<<"vecRef2transv[0] "<<vecRef2transv[0] <<endl;
-    cout<<"vecRef2transv[1] "<<vecRef2transv[1] <<endl;
-    cout<<"vecRef2transv[2] "<<vecRef2transv[2] <<endl;
-  */
 
   vecRef1transv *= 1/vecRef1transv.Mag();
   vecRef2transv *= 1/vecRef2transv.Mag();
 
   double acop = TMath::ACos(vecRef1transv*vecRef2transv);
-  
+  double acoporiginal=acop;
   double sign = vecPi2 * vecRef1transv.Cross(vecRef2transv);
+  double psioriginal =sign;//
```

Comment:
I can't find where this variable is used, would you mind pointing it to me?

Reply:
Hi it is not used anymore, obsolete. May be deleted. Indeed, I should make more often small pull requests...